docs/source/_toctree.yml

```diff
@@ -60,7 +60,7 @@
   - local: debugging
     title: Debugging
   - local: serialization
-    title: Exporting transformers models
+    title: Exporting 🤗 Transformers models
```

Comment:
I took the liberty of updating this title to match the convention used in the rest of the ToC